src/cmd/scan.go

```diff
@@ -0,0 +1,60 @@
+/*
+Copyright © 2025 NAME HERE <EMAIL ADDRESS>
+*/
```

Comment:
Let's be sure to use the standard SPDX head matter on these files

```suggestion
// SPDX-License-Identifier: Apache-2.0
// SPDX-FileCopyrightText: 2025-Present Defense Unicorns Engineering```

Reply:
fixed in next commit